registries/adapters/dockerhub_adapter.go

```diff
@@ -222,24 +220,30 @@ func (r DockerHubAdapter) loadSpec(imageName string) (*bundle.Spec, error) {
 	if r.Config.Tag == "" {
 		r.Config.Tag = "latest"
 	}
-	req, err := http.NewRequest("GET", fmt.Sprintf(dockerHubManifestURL, imageName, r.Config.Tag), nil)
+
+	token, err := r.getBearerToken(imageName)
 	if err != nil {
 		return nil, err
 	}
-	token, err := r.getBearerToken(imageName)
+
+	req, err := http.NewRequest("GET", fmt.Sprintf(dockerHubManifestURL, imageName, r.Config.Tag), nil)
 	if err != nil {
 		return nil, err
 	}
+
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", token))
 	req.Header.Add("Accept", "application/json")
+
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
 		return nil, err
 	}
+
 	body, err := registryResponseHandler(resp)
 	if err != nil {
 		return nil, fmt.Errorf("DockerHubAdapter::error handling dockerhub registery response %s", err)
 	}
+	fmt.Printf("body: [%v], %s\n", string(body), fmt.Sprintf("%s/%s:%s", r.RegistryName(), imageName, r.Config.Tag))
```

Comment:
Should we log this at debug level? or should we just remove it?

Reply:
+1 I think this will play ugly with the `apb` tool.